aten/src/ATen/nn_parse.py

```diff
@@ -235,7 +235,7 @@ def function_info(name, arguments, cimpls, buffers, backends, inplace, scalar_ch
     return {
         'mode': 'NN',
         'name': name,
-        'types': ['Float', 'Double', 'Half'],  # Half will be stripped for CPU backend
```

Comment:
why did you strip the Half comment?

Reply:
Moved: https://github.com/pytorch/pytorch/pull/20729/files#diff-5fa7dc4c0a84674c4d9b12cd706740f9R418

Reply:
Moved: https://github.com/pytorch/pytorch/pull/20729/files#diff-5fa7dc4c0a84674c4d9b12cd706740f9R418